### PR TITLE
Added Logo and Splashscreen.

### PR DESCRIPTION
**Pull request by Vrushti24**

### CODE REVIEW

**Review by anushbhatia — CHANGES_REQUESTED** [review]
Great work with the home screen @Vrushti24🙌🙌🙌 but don't modify the side nav as it is not working now please resolve that rest everything is working fine.

**Comment by Vrushti24** [comment]
Can u explain me in detail what is not working on side nav because I
changed only logo nothing else and I checked that It was working fine so
please let me know in detail about the issue
And thank u .

File: lib/main.dart [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
-  _MyAppState createState() => _MyAppState();
-}
-
-class _MyAppState extends State<MyApp> {
+class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      home: SplashScreen(),
+      debugShowCheckedModeBanner: false,
```
Comment (anushbhatia) [inline lib/main.dart:17]: Don't modify the icon I will provide the nav icon soon as it does currently not function.

**Comment by Vrushti24** [comment]
Okay great I will do that

**Comment by Vrushti24** [comment]
I will remove the icon from maindrawer.dart ?

**Comment by anushbhatia** [comment]
https://user-images.githubusercontent.com/40017559/103419850-c1104000-4bba-11eb-898b-787cf1921b90.mp4

**Comment by Vrushti24** [comment]
I got it i will change that thanks

**Comment by Vrushti24** [comment]
I have made the changes and even side nav is also working now how can I add
this to GitHub again Same process like before ?

**Comment by Vrushti24** [comment]
I have updated PR #16 please check it if this is properly updated thank u

**Comment by Vrushti24** [comment]
@anushbhatia I hope that the issue is solved U can check now PR  #16

**Comment by Vrushti24** [comment]
Thank you!

**Pull request merged**